_data/info.yml

```diff
@@ -0,0 +1,22 @@
+contacts:
+  - title: Twitter
+    url: https://twitter.com/ruby_nepal/
```

Comment:
`twitter_username` is repeated here and the data from config can be reused.

Reply:
`_data/info.yml` is outdated now.